seed_r7_bringup/launch/seed_r7_bringup.launch

```diff
@@ -26,6 +26,12 @@
       <arg name="load_robot_description" value="true"/>
     </include>
   </group>
+  <group if="$(eval robot_model== 'typeFCETy')">
```

Comment:
There is no model and csv data. 
in future, will you update or mistake ?

Reply:
I deleted about typeFCET,. When next version is  completed,  will update typeFCET.